src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -0,0 +1,1882 @@
+		for (EventListener listener : listeners) {
+			((InventoryListener) listener).InventoryInserted(event);
+		}
+		jTableInventoryRow.updateUI();
+	}
+
+	private GoodDateChooser jCalendarInventory;
+	private LocalDateTime dateInventory = TimeTools.getServerDateTime();
+	private JPanel panelHeader;
+	private JPanel panelFooter;
+	private JPanel panelContent;
+	private JButton closeButton;
+	private JButton saveButton;
+	private JButton resetButton;
+	private JButton lotButton;
+	private JButton printButton;
+	private JButton validateButton;
+	private JButton deleteButton;
+	private JScrollPane scrollPaneInventory;
+	private JTable jTableInventoryRow;
+	private JButton confirmButton;
+	private List<MedicalInventoryRow> inventoryRowList = new ArrayList<>();
+	private List<MedicalInventoryRow> inventoryRowSearchList = new ArrayList<>();
+	private List<MedicalInventoryRow> inventoryRowsToDelete = new ArrayList<>();
+	private List<MedicalInventoryRow> inventoryRowListAdded = new ArrayList<>();
+	private List<Lot> lotsSaved = new ArrayList<>();
+	private HashMap<Integer, Lot> lotsDeleted = new HashMap<>();
+	private String[] pColums = { MessageBundle.getMessage("angal.inventory.id.col").toUpperCase(),
+			MessageBundle.getMessage("angal.common.code.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.medical.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.newlot.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.lotcode.col").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.duedate.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.theorticalqty.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.realqty.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.unitprice.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.totalprice.col").toUpperCase()
+	};
+	private int[] pColumwidth = { 50, 50, 200, 100, 100, 100, 100, 80, 80, 80 };
+	private boolean[] columnEditable = { false, false, false, false, false, false, false, true, false, false };
+	private boolean[] columnEditableView = { false, false, false, false, false, false, false, false, false, false };
+	private boolean[] pColumnVisible = { false, true, true, true, !GeneralData.AUTOMATICLOT_IN, true, true, true, GeneralData.LOTWITHCOST,
+			GeneralData.LOTWITHCOST };
+	private final String[] lotSelectionColumnNames = {
+			MessageBundle.getMessage("angal.medicalstock.lotid").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.prepdate.col").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.duedate").toUpperCase()
+	};
+	private final Class[] lotSelectionColumnClasse = { String.class, String.class, String.class };
+	private MedicalInventory inventory = null;
+	private JRadioButton specificRadio;
+	private JRadioButton allRadio;
+	private JLabel dateInventoryLabel;
+	private JTextField codeTextField;
+	private String code = null;
```

Comment:
```suggestion
	private String code;
```
assigning `null` is redundant